test/network/2020-03-01/armnetwork/vpnsites.go

```diff
@@ -161,11 +164,15 @@ func (client *vpnSitesOperations) ResumeDelete(token string) (HTTPPoller, error)
 
 // deleteCreateRequest creates the Delete request.
 func (client *vpnSitesOperations) deleteCreateRequest(resourceGroupName string, vpnSiteName string) (*azcore.Request, error) {
+	u, err := url.Parse(client.u)
+	if err != nil {
+		return nil, err
+	}
 	urlPath := "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnSites/{vpnSiteName}"
 	urlPath = strings.ReplaceAll(urlPath, "{subscriptionId}", url.PathEscape(client.subscriptionID))
 	urlPath = strings.ReplaceAll(urlPath, "{resourceGroupName}", url.PathEscape(resourceGroupName))
 	urlPath = strings.ReplaceAll(urlPath, "{vpnSiteName}", url.PathEscape(vpnSiteName))
-	u, err := client.u.Parse(path.Join(client.u.Path, urlPath))
+	u, err = u.Parse(urlPath)
```

Comment:
Just double checking here, this will append urlPath to the already existing path on the url?

Reply:
Good catch, no it doesn't work as expected.  Will fix.

Reply:
Fixed